### ISSUE TITLE
[1.16] Applying model override to item model causes enchant glint to render solid (?)

### ISSUE THREAD

**Issue by ChloeDawn** [Chisel-Team/ConnectedTexturesMod#130]
**Mod version:** [MC1.16.1-1.1.0.1](https://www.curseforge.com/minecraft/mc-mods/ctm/files/2988953)

Tested with `SOLID` and `TRANSLUCENT` layer, resulting in the same issue. GUI model seems to be fine, and using the approach of a dedicated texture with texture metadata does not cause this issue.

![image](https://user-images.githubusercontent.com/9869940/88463907-31998f00-ceae-11ea-895d-2d098f030ee1.png)
```json
{
	"parent": "minecraft:item/generated",
	"textures": {
		"layer0": "minecraft:item/golden_apple"
	},
	"loader": "ctm:ctm",
	"ctm_version": 1,
	"ctm_overrides": {
		"0": {
			"type": "normal",
			"layer": "TRANSLUCENT",
			"extra": {
				"light": {
					"block": 15
				}
			}
		}
	}
}
```

An implementation using a dedicated texture and texture metadata that renders correctly:

![image](https://user-images.githubusercontent.com/9869940/88464002-0d8a7d80-ceaf-11ea-8ea2-bf25ab05378a.png)

**Comment by ChloeDawn** [comment]
On closer inspection, the texture is still there, just the glint is significantly more opaque than it should be.

**Comment by ChloeDawn** [comment]
It appears to be sensitive to world light. 

![image](https://user-images.githubusercontent.com/9869940/88463980-d61bd100-ceae-11ea-99be-9b7cfe0b9cb6.png)

**Comment by tterrag1098** [comment]
This issue manifested slightly differently in 1.15, but I believe it's the same problem. If this ends up still broken after being merged into 1.16, please update this issue.
